README.md

```diff
@@ -12,17 +12,10 @@ Run:
 
 ```
 npm i -g yarn
-yarn global add expo-cli
+npm i -g expo-cli
+npm i -g eas-cli
```

Comment:
What is `eas-cli`?

Reply:
`eas-cli` is the CLI for [Expo Application Services](https://docs.expo.dev/build/introduction/). It's the service expo provides to build our app for us

Reply:
Cool, thanks!